pcdsdevices/delay_generator.py

```diff
@@ -0,0 +1,141 @@
+"""
+Delay generator class
+
+This module contains classes related to the SRS DG645 delay generator.
+"""
+
+from time import sleep
+
+from ophyd import Device
+from ophyd import Component as Cpt
+from ophyd import EpicsSignalRO
+from ophyd import EpicsSignal
+
+from pcdsdevices.interface import BaseInterface
+
+
+CHANNELS = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'T0']
+TRIGGER_SOURCES = {
+    '0': 'Internal',
+    '1': 'Ext ^edge',
+    '2': 'Ext ~edge',
+    '3': 'SS ext ^edge',
+    '4': 'SS ext ~edge',
+    '5': 'Single Shot',
+    '6': 'Line'
+}
+TRIGGER_INHIBITS = {
+    '0': 'Off',
+    '1': 'Triggers',
+    '2': 'AB',
+    '3': 'AB,CD',
+    '4': 'AB,CD,EF',
+    '5': 'AB,CD,EF,GH'
+}
+
+
+class Dg_channel(BaseInterface, Device):
+    """
+    Delay generator single channel class
+
+    Parameters
+    ----------
+    prefix: str
+        Base PV for the delay generator channel
+
+    name: str
+        Alias of the channel
+    """
+    delay = Cpt(EpicsSignal, 'DelayAO', kind='hinted')
```

Comment:
Usually we include the colon in the component suffix to make things a little cleaner when instantiating the device, e.g. `delay = Cpt(EpicsSignal, ':DelayAO', kind='hinted')`

Reply:
Nevermind, I see you handle this below. Don't mind me....